deployment/ansible/deploy.yml

```diff
@@ -168,6 +166,24 @@
         when: genesis_accounts is defined
         tags: [config]
 
+      - name: "Copy public peer key"
```

Comment:
Please use similar (same/copied) [task that deploys the chain keys](https://github.com/aeternity/epoch/pull/985/files#diff-2ad1947fe9b869ffd23a6a86872e65e0R146)

- you don't need user/group as it's already executing with the correct user
- mode should be `0600` instead of `0644`, otherwise it's visible to all users
- can be combined in one task thanks to `with_items:` - see above

Reply:
644 was there on nodes with current files, thats why put 633

Reply:
I guess we have to change the default mask at some point (additional ticket .. )